cmd/create/network/cmd.go

```diff
@@ -157,5 +160,11 @@ func extractTemplateCommand(r *rosa.Runtime, argv []string, options *opts.Networ
 	} else {
 		templateDir := options.TemplateDir
 		*templateFile = helper.SelectTemplate(templateDir, *templateCommand)
+		templateFile, err := os.ReadFile(*templateFile)
```

Comment:
Is it intentional to hide the param with a new var?

Reply:
Yes because I cant reassign the templateFile pointer in this scenario